src/handler.py

```diff
@@ -181,3 +196,206 @@ def _stop_db(db, triggers):
             stopped = True
     return stopped
 
+
+def modify_postgres_password(event, context):
+    _validate()
+    logger.info("enter modify postgres password")
+    logger.info(event)
+    original = secrets_client.get_secret_value(
+        SecretId=NWCAPTURE_REAL,
+    )
+    secret_string = json.loads(original['SecretString'])
+    postgres_password = secret_string['POSTGRES_PASSWORD']
+
+    response = rds_client.describe_db_clusters()
+    logger.info(f" all clusters {response}")
+    rds_client.modify_db_cluster(
+        DBClusterIdentifier=DEFAULT_DB_CLUSTER_IDENTIFIER,
+        ApplyImmediately=True,
+        MasterUserPassword=postgres_password
+    )
+
+
+def delete_capture_db(event, context):
+    _validate()
+    try:
+        rds_client.delete_db_instance(
+            DBInstanceIdentifier=DEFAULT_DB_INSTANCE_IDENTIFIER,
+            SkipFinalSnapshot=True
+        )
+    except rds_client.exceptions.DBInstanceNotFoundFault as e:
+        """
+        We could be in a messed up state where the instance doesn't exist but the cluster does,
+        due to vagaries of how long AWS takes to set up a cluster, so proceed
+        """
+
+    rds_client.delete_db_cluster(
+        DBClusterIdentifier=DEFAULT_DB_CLUSTER_IDENTIFIER,
+        SkipFinalSnapshot=True
+    )
+    disable_triggers(TRIGGER[os.environ['STAGE']])
+
+
+def create_db_instance(event, context):
+    _validate()
+    logger.info("enter create db instance")
+    logger.info(event)
+    stage = os.environ['STAGE'].lower()
+    rds_client.create_db_instance(
+        DBInstanceIdentifier=DEFAULT_DB_INSTANCE_IDENTIFIER,
+        DBInstanceClass=DEFAULT_DB_INSTANCE_CLASS,
+        DBClusterIdentifier=DEFAULT_DB_CLUSTER_IDENTIFIER,
+        Engine=ENGINE,
+        Tags=[
+            {'Key': 'Name', 'Value': f"NWISWEB-CAPTURE-RDS-AURORA-{stage.upper()}"},
+            {'Key': 'wma:applicationId', 'Value': 'NWISWEB-CAPTURE'},
+            {'Key': 'wma:contact', 'Value': 'tbd'},
+            {'Key': 'wma:costCenter', 'Value': 'tbd'},
+            {'Key': 'wma:criticality', 'Value': 'tbd'},
+            {'Key': 'wma:environment', 'Value': stage},
+            {'Key': 'wma:operationalHours', 'Value': 'tbd'},
+            {'Key': 'wma:organization', 'Value': 'tbd'},
```

Comment:
I would set organization to `IOW`.

Reply:
Okay.  It may cause some transitional confusion in the cloudcheckr reports, but I agree in the long run it's better.